wmn-data.json

```diff
@@ -5455,6 +5527,30 @@
         "cat" : "blog",
         "valid" : true
        },
+       {
+        "name" : "StreamElements",
+        "uri_check" : "https://api.streamelements.com/kappa/v2/channels/{account}",

```

Comment:
This is a great check_uri. I'm wondering if there is a link to the user's profile that we can add in the `pretty_uri` field to take users right to it instead of to this JSON